serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc

```diff
@@ -3,17 +3,53 @@
+:openshift_operator_install_url: https://docs.openshift.com/container-platform/4.13/operators/admin/olm-adding-operators-to-cluster.html
+:openshift_operator_uninstall_url: https://docs.openshift.com/container-platform/4.13/operators/admin/olm-deleting-operators-from-cluster.html
+:kubernetes_operator_install_url: https://operatorhub.io/how-to-install-an-operator
+:kubernetes_operator_uninstall_url: https://olm.operatorframework.io/docs/tasks/uninstall-operator/
+:operatorhub_url: https://operatorhub.io/
+
+This guide describes how to install the {operator_name} in a Kubernetes or OpenShift cluster. The operator is in an xref:/cloud/operator/known-issues.adoc[early development stage] (community only) and has been tested on OpenShift 4.11+, Kubernetes 1.22+, and link:{minikube_url}[Minikube].
 
 .Prerequisites
-* A Kubernetes cluster with admin privileges. Alternatively, you can use Minikube or KIND.
+* A Kubernetes or OpenShift cluster with admin privileges. Alternatively, you can use Minikube or KIND.
 * `kubectl` command-line tool is installed. Otherwise, Minikube provides it.
 
-== Prepare a Minikube Instance
+== {product_name} Operator OpenShift Installation
+
+To install the operator on OpenShift follow the document "link:{openshift_operator_install_url}[Adding Operators to a cluster]" from the OpenShift's documentation.
+
+When searching for the operator in the *Filter by keyword* field, use the word `{operator_openshift_keyword}`. If you're installing from the CLI, the operator's catalog name is `{operator_openshift_catalog}`.
+
+=== Uninstall
+
+To remove the operator on OpenShift follow the document "link:{openshift_operator_uninstall_url}[Deleting Operators from a cluster]" from the OpenShift's documentation.
+
+== {product_name} Operator Kubernetes Installation
+
+To install the operator on Kubernetes follow the document "link:{kubernetes_operator_install_url}[How to install an Operator from OperatorHub.io]" from the OperatorHub's documentation.
+
+When link:{operatorhub_url}[searching for the operator in the *Search OperatorHub* field], use the word `{operator_k8s_keyword}`.
+
+=== Uninstall
+
+To remove the operator on Kubernetes follow the document "link:{kubernetes_operator_uninstall_url}[Uninstall your operator]" from the OLM's documentation.
+
+When searching for the subscription to remove, use the word `{operator_k8s_subscription}`.
+
+== {product_name} Operator Manual Installation
 
 [WARNING]
 ====
+If you're running on Kubernetes or OpenShift, It's highly recommended to install the operator from the OperatorHub or OpenShift Console instead since the installation is managed by OLM. Use this method only if you need a snapshot version or you're running locally on Minikube or KIND.
```

Comment:
```suggestion
If you're running on Kubernetes or OpenShift, it is highly recommended to install the operator from the OperatorHub or OpenShift Console instead since the installation is managed by OLM. Use this method only if you need a snapshot version or you're running locally on Minikube or KIND.
```